carma-platform.repos

```diff
@@ -0,0 +1,81 @@
+repositories:
+  CARMAAvtVimbaDriver:
+    type: git
+    url: git@github.com:usdot-fhwa-stol/CARMAAvtVimbaDriver.git
+    version: develop
```

Comment:
You may have described to me, but I am not remembering what does version: develop mean?  I see that Autoware.ai and the Novatel driver are set to master, but everything else is develop.  Are the rules for this value documented somewhere?

Reply:
In this case, when you use `vcstool` to checkout all these repos at once, this one is checked out to develop by default. I think for all these repos it's just using the default branch as defined in github, the one that default PRs are against, so this behavior makes sense to me.

Reply:
Then we probably need to fix the Novatel driver repo, since it's showing master here.

Reply:
Yep, it was set with master as default.  I have fixed that.  Please make the corresponding change in this file.

Reply:
I guess I should do the same thing on autoware.ai fork as well, since we don't want to be branching/pushing directly from master as a rule.  So please change both of those here.

Reply:
I actually updated the autoware.ai repo to default to the carma-develop branch.